configs/smartling-config.json

```diff
@@ -1,8 +1,24 @@
 {
     "locales": [{
+            "smartling": "ar",
+            "application": "ar"
+        },      
+        {
             "smartling": "ms-MS",
             "application": "ms"
-        }
+        },
+        {
+            "smartling": "hi-IN",
+            "application": "hi-IN"
+        },
+        {
+            "smartling": "id-ID",
+            "application": "id"
+        },
+        {
+            "smartling": "ja-JP",
+            "application": "ja"
+        },                
```

Comment:
Remove the comma here, to indicate this is the last one. Or does it matter? This should be the last one!